<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>File</class>
 <widget class="QDialog" name="File">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>580</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>751</width>
     <height>581</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page_5">
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btnShare">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>93</width>
       <height>29</height>
      </rect>
     </property>
     <property name="text">
      <string>share</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_6">
    <widget class="QLineEdit" name="txtAge">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>340</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtDateofBirth">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>410</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>260</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Contact Number:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>340</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Email Address:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>410</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Date Donated:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>260</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Last Name:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtEmailAddress">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>340</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtDateRequested">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>410</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtLastName">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>260</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtOccupation">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>190</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtAddress">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>120</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtBloodTypeNeeded">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>490</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>490</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Gender:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>190</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Occupation:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtContactNumber">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>260</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>190</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Middle Name:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtMiddleName">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>190</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>120</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>First Name:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>60</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>ID:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtGender">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>490</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>120</y>
       <width>131</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Area of Residence:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>410</y>
       <width>91</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Date of Birth:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtFirstName">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>120</y>
       <width>171</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>340</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Age:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="txtCID">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>60</y>
       <width>113</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>490</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Blood Type:</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
